spec/datadog/profiling/component_spec.rb

```diff
@@ -1006,13 +1013,8 @@
     context "when no_signals_workaround_enabled is an invalid value" do
       before do
         settings.profiling.advanced.no_signals_workaround_enabled = "invalid value"
-        allow(Datadog.logger).to receive(:error)
-      end
 
-      it "logs an error message mentioning that the invalid value will be ignored" do
-        expect(Datadog.logger).to receive(:error).with(/Ignoring invalid value/)
-
-        no_signals_workaround_enabled?
+        expect(logger).to receive(:warn).with(/Ignoring invalid value/)
       end
 
       include_examples "no_signals_workaround_enabled :auto behavior"
```

Comment:
I chose to simplify this a bit because the change from logging this info at error to warn was also interacting with a few of the specs from the `include_examples`, so I chose to just call the `expect` in the before rather than doing something weirder (or copy-pasting the specs to change them slightly).